data/editor.config

```diff
@@ -557,6 +557,10 @@
         <include group="poi_internet" />
         <include field="wikipedia" />
       </type>
+
+      <!-- Do not allow to edit settlements.
+           It is not just about editing names, because of the complex OSM address scheme.
```

Comment:
What is broken now when user edits the village?

I would leave editing alternate languages.

Reply:
Towns, and villages are something that already have valid names or fixed by a community.
For example, addr:place is referenced by name. Generator transforms relations, etc.

We have one edit example and it is not right. Almost sure, there are no other correct village name editings.
Score - 100% bad for now.